### PR TITLE
Added jupyter python script 

### PR DESCRIPTION
**Pull request by Ayesha279**
### Description
Added jupyter python script 
### Fixes
New file called OCM-benchmark test added

### CODE REVIEW

File: src/touchstone/OCM-benchmark test(1).py [hunk 1]
```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
```
Comment (jtaleric) [inline src/touchstone/OCM-benchmark test(1).py:1]: lets create a new repository for juypter, lets not have it in benchmark-comparison.

File: src/touchstone/databases/elasticsearch.py [hunk 2]
```diff
@@ -71,8 +71,9 @@ def emit_compute_dict(self, uuid, compute_map, index, identifier):
 
         logger.debug("Initializing search object")
         kw_identifier = identifier + ".keyword"  # append .keyword
-        s = Search(using=self._conn_object, index=str(index)).query("match", **{kw_identifier: uuid})
-
+        #s = Search(using=self._conn_object, index=str(index)).query("match", **{kw_identifier: uuid})
+         s= elasticsearch.helpers.scan.(self._conn_object, query={"query": {"match", **{kw_identifier: uuid}}}, index=str(index), scroll='1m')
```
Comment (jtaleric) [inline src/touchstone/databases/elasticsearch.py:75]: spacing seems off, additionally - does this break any previous queries?

File: src/touchstone/OCM-benchmark test(1).py (hunk 1 above)
Comment (Ayesha279) [reply src/touchstone/OCM-benchmark test(1).py:1]: Okay, got it

File: src/touchstone/databases/elasticsearch.py (hunk 2 above)
Comment (Ayesha279) [reply src/touchstone/databases/elasticsearch.py:75]: Forgot to comment the first 's' out. Making a few for changes to the code, will put it in.

**Comment by jtaleric** [comment]
Is this still relevant?

**Pull request closed without merging**